### PR TITLE
Fix Filter deserialization

### PR DESCRIPTION
**Pull request by 9il**

### CODE REVIEW

File: source/jmap/types.d [hunk 1]
```diff
@@ -898,6 +903,15 @@ struct Filter {
     }
 }
 
+// checks de/serialization compiles
+unittest
+{
+    import mir.ser.text;
+    import mir.deser.text;
+    assert(Filter.init.serializeText == `null`);
```
Comment (kinke) [inline source/jmap/types.d]: Thx, the compile error is gone now. - I get `null.struct` here.

Comment (9il) [reply source/jmap/types.d]: I have changed it to JSON

Comment (kinke) [reply source/jmap/types.d]: That yields `{}`.

Comment (9il) [reply source/jmap/types.d]: haha, I forgot how it works...

**Pull request merged**